test/Microsoft.AspNet.Authentication.Test/DataHandler/TicketSerializerTests.cs

```diff
@@ -6,7 +6,6 @@
 using System.Linq;
 using System.Security.Claims;
 using Microsoft.AspNet.Http.Authentication;
-using Shouldly;
```

Comment:
Unrelated to this PR, do we want to nuke Shouldly from Authentication tests entirely at some point?